spec/index.md

```diff
@@ -2114,6 +2114,24 @@ This function returns fee percentiles, measured in millisatoshi/vbyte (1000 mill
 
 The [standard nearest-rank estimation method](https://en.wikipedia.org/wiki/Percentile#The_nearest-rank_method), inclusive, with the addition of a 0th percentile is used. Concretely, for any i from 1 to 100, the ith percentile is the fee with rank `⌈i * 100⌉`. The 0th percentile is defined as the smallest fee (excluding coinbase transactions).
 
+### IC method `metrics` {#ic-metrics}
+
+:::note
+
+The metrics management canister API is considered EXPERIMENTAL. Canister developers must be aware that the API may evolve in a non-backward-compatible way.
+
+:::
+
+Given a subnet ID as input, this method returns a collection of metrics for the given subnet (field `metrics`) and a timestamp provided in nanoseconds since 1970-01-01 (field `timestamp_nanos`) at which the metrics are sampled.
+
+A single metric entry is of an enumeration type with a single variant (in the future, more variants might be added):
+
+- `node_metrics`: provides metrics for a node characterized by its principal (field `node_id`) and the actual metrics values:
+
+  - `num_blocks_total` (`nat64`): the number of blocks proposed by this node;
+
+  - `num_block_failures_total` (`nat64`): the number of failed block proposals by this node.
+
```

Comment:
```suggestion
When calling `metrics(subnet_id, minimal_timestamp_nanos)` the response returns the smallest timestamp > 'minimal_timestamp_nanos' for which metrics are stored in replicated state, together with a `node_metrics` entry for each node of the subnet.
```

Reply:
`>=` instead of `>`?

Reply:
yes, that makes more sense

Reply:
I'd have the following suggestion: let's add the sentence
```
This timestamp is the earliest timestamp after the requested timestamp (field `requested_timestamp_nanos`) for which the metrics are available.
```
after the existing sentence
```
Given a subnet ID as input, this method returns a collection of metrics for the given subnet (field `metrics`) and a timestamp provided in nanoseconds since 1970-01-01 (field `timestamp_nanos`) at which the metrics are sampled.
```

Reply:
Sounds great, thanks for the suggestion!